dts/bindings/sensor/adi,adxl345-common.yaml

```diff
@@ -1,27 +1,41 @@
 # Copyright (c) 2022 Analog Devices Inc.
 # SPDX-License-Identifier: Apache-2.0
 
+description: |
+  ADXL345 3-axis accelerometer
+  When setting the accelerometer DTS properties, make sure to include
+  adxl345.h and use the macros defined there.
+
+  Example:
+  #include <zephyr/dt-bindings/sensor/adxl345.h>
+
+  adxl345: adxl345@1d {
+    ...
+
+    odr = <ADXL345_DT_ODR_25>;
+  };
+
 include: sensor-device.yaml
 
 properties:
   odr:
     type: int
-    default: 0
+    default: 10
     description: |
           Accelerometer sampling frequency (ODR). Default is power on reset value.
-            0 # 12.5Hz
-            1 # 25Hz
-            2 # 50Hz
-            3 # 100Hz
-            4 # 200Hz
-            5 # 400Hz
     enum:
-      - 0
-      - 1
-      - 2
-      - 3
-      - 4
-      - 5
+      - 7 # ADXL345_DT_ODR_12_5
+      - 8 # ADXL345_DT_ODR_25
+      - 9 # ADXL345_DT_ODR_50
+      - 10 # ADXL345_DT_ODR_100
+      - 11 # ADXL345_DT_ODR_200
+      - 12 # ADXL345_DT_ODR_400
+
+  fifo-watermark:
```

Comment:
We should pick a direction and go with it. Most applications I've worked with (usually in AR/VR and Android) expect that the scheduling of the processing thread is predictable. So I much prefer to use SENSOR_ATTR_BATCH_DURATION or in DT some form of `batch-duration-us`. This way you can expect to get samples at the same time interval regardless of the ODR.

Whatever direction we go we should be consistent.

Reply:
Friendly reminder @yperess: please let me know if there's anything else required for removing the NACK.

Thanks in advance!